src_new/constants.js

```diff
@@ -163,3 +163,5 @@ exports.HOOKS = {
 	"BID_RECEIVED": "HookForBidReceived",
 	"POST_AUCTION_KEY_VALUES": "HookForPostAuctionKeyValues"
 };
+
+exports.BIDDER_NAMES = ["rubicon","improvedigital"];
```

Comment:
"BIDDER_NAMES" name is not intuitive with the feature/usage in mind
rename it to SRA_ENABLED_BIDDERS = { "rubicon": 1, "improvedigital": 1 };
now you will not need to check in an array by looping, just check if bidder exists in the object

Reply:
Done

Reply:
I still see the SRA_ENABLED_BIDDERS as an array, the above comment mentions to change it to an object. Also, related changes are not done which can avoid the need for partnerToEnableSRA array.

Reply:
My Bad regarding object 
Changed by converting it into object.

Reply:
yes, or you can use 
config.forEachAdapter(function(adapter){
    if(constants.sra_enabled_bidders.hasOwnProperty(adapter)){
       // addd extra config in prebidConfig at appropriate location
    }
})

Reply:
even reverse loop is better.

Reply:
Yeah 
Updated the code